sled-agent/src/bootstrap/rack_ops.rs

```diff
@@ -171,9 +179,14 @@ impl RssAccess {
             RssStatus::Uninitialized { .. } => {
                 let (completion_tx, completion) = oneshot::channel();
                 let id = RackInitUuid::new_v4();
-                *status = RssStatus::Initializing { id, completion };
+                let (step_tx, step_rx) = watch::channel(RssStep::Requested);
+                *status = RssStatus::Initializing {
+                    id,
+                    completion,
+                    step_rx,
+                    step: RssStep::Requested,
```

Comment:
Tiny nit - I think this could be

```suggestion
                    step: *step_rx.borrow(),
```

to avoid needing to duplicate `RssStep::Requested`

Reply:
done